install

```diff
@@ -379,10 +379,12 @@ Dir.chdir HOMEBREW_REPOSITORY do
 
   system git, "reset", "--hard", "origin/master"
 
-  unless can_macos_use_tls12?
+  if can_macos_use_tls12?
```

Comment:
This is not a correct change; this is making modern versions of macOS use `CORE_TAP_REPO_INSECURE`

Reply:
Is this change ok?
```suggestion
  if can_macos_use_tls12?
    system git, "clone", "--depth", "1", CORE_TAP_REPO, HOMEBREW_CORE_TAP
  else
    system git, "clone", "--depth", "1", CORE_TAP_REPO_INSECURE, HOMEBREW_CORE_TAP
  end

```

Reply:
It's not necessary as it's done by Homebrew itself.